src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/services/reports/ReportsServiceTest.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.services.reports;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.reports.EnvelopeCountSummaryRepository;
+import uk.gov.hmcts.reform.bulkscanprocessor.helper.reports.countsummary.Item;
+
+import java.util.List;
+
+import static java.time.LocalDate.now;
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ReportsServiceTest {
+
+    @Mock
+    EnvelopeCountSummaryRepository repo;
+
+    @Test
+    public void should_map_repo_result_properly() {
```

Comment:
To consider: a case when the repo returns an empty list. Up to you.

Reply:
added